data/Templates/Hl7v2/Resource/_PractitionerRole.liquid

```diff
@@ -34,7 +34,7 @@
         [
             {
                 {% if ROL.13 -%}
-                "reference":"Location/{{generateUUID ROL-13}}",
+                "reference":"Location/{{ Location_ID_ROL_13 }}",
```

Comment:
Currently PractitionerRole is never generated from our root templates, right?

Reply:
Yes, currently we are not using it, we will update the main template to include more resources.

Reply:
Got it. Just to double confirm if I understand it correctly. Thanks!